tests/sqlparser_mssql.rs

```diff
@@ -283,6 +294,50 @@ fn parse_create_function() {
         END\
     ";
     let _ = ms().verified_stmt(create_function_with_return_expression);
+
+    let create_inline_table_value_function = "\
+        CREATE FUNCTION some_inline_tvf(@foo INT, @bar VARCHAR(256)) \
+        RETURNS TABLE \
+        AS \
+        RETURN (SELECT 1 AS col_1)\
```

Comment:
Parentheses are optional for inline tvf `return` queries, although I think the subquery expr expects/requires them currently.

Reply:
I added support for that syntax & added a new test case example

Reply:
`UNION` is also supported in this syntax, but I'm comfortable leaving that for later